powerauth-backend-tests/src/test/resources/application.properties

```diff
@@ -19,6 +19,9 @@ powerauth.service.security.clientSecret=
 powerauth.test.application.name=PA_Tests
 powerauth.test.application.version=default
 
+# Set whether to include custom tests, such as onboarding tests and test requiring specific customizations in enrollment-server-wultra
+powerauth.test.includeCustomTests=false
```

Comment:
🤔 I am thinking about parameterizing from outside via ENVs, but I do not see it elsewhere anyway.

Reply:
Aha, it is because Spring Boot maps it out of the box from `POWERAUTH_TEST_INCLUDECUSTOMTESTS`. Moreover, do we prefer `powerauth.test.includeCustomTests` or `powerauth.test.include-custom-tests`?

Reply:
This can be mapped easily. I followed the convention used for other properties (last part is camel case), if we want to change it, we should change all other properties as well.